Solutions/Windows Security Events/Hunting Queries/persistence_create_account.yaml

```diff
@@ -1,6 +1,8 @@
 id: 5e76eaf9-79a7-448c-bace-28e5b53b8396
 name: Summary of users created using uncommon/undocumented commandline switches
 description: |
+  'Summarizes uses of uncommon & undocumented commandline switches to create persistence. User accounts may be created to achieve persistence on a machine.'
```

Comment:
Summarizes uses of uncommon & undocumented commandline switches to create user accounts. User accounts may be created to achieve persistence on a machine.